docs/layers/accounts/tutorials/how-to-set-up-aws-email-notifications.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "How to Set Up AWS Email Notifications"
+confluence: https://cloudposse.atlassian.net/wiki/spaces/REFARCH/pages/1176371445/How+to+Set+Up+AWS+Email+Notifications
+sidebar_position: 100
+custom_edit_url: https://github.com/cloudposse/refarch-scaffold/tree/main/docs/docs/setup/cold-start/how-to-set-up-aws-email-notifications.md
+---
+
+# How to Set Up AWS Email Notifications
+
+:::caution Make sure to use the email address corresponding to the
+[Decide on Email Address Format for AWS Accounts](/reference-architecture/fundamentals/design-decisions/cold-start/decide-on-email-address-format-for-aws-accounts)
+ADR.
+
+:::
+
```

Comment:
Weird admonition. The title is like a sentence.

Reply:
prettier formatted these incorrectly. I'll fix